src/user/user.controller.ts

```diff
@@ -43,16 +44,24 @@ export class UserController {
     if (!req.user || !req.body.email) {
```

Comment:
This shouldn't happen - if the execution reaches this point (having the JWT auth guard), it means the request is authenticated

Reply:
fixed by https://github.com/alkemyTech/CENCO-MIT-Back-T1/pull/27/commits/200b86b0c16713c23653e196873ac7c0f5ad3ba3